### ISSUE TITLE
Compatibility of dia-PASEF window design and DIA-NN analysis

### ISSUE THREAD

**Issue by joachims1956** [vdemichev/DiaNN#1160]
Dear Demichev, 
Thanks for this amazing software. 
We were wondering whether there are some limitations for the dia-PASEF window design of TimsTOF instrument regarding DIA-NN analysis. 
We failed to analyze samples acquired with this window design (please see screenshot below) in DIA-NN 1.9.1. Are the any "Additional options" to enable analysis in DIA-NN? 
Thank you very much. 

![image](https://github.com/user-attachments/assets/787fbc20-47d6-4b18-af23-5a856e353fd8)

**Comment by vdemichev** [comment]
Hi Joachim,

Please try if it works with --tims-scan.

What is the rationale behind this design? If the idea is to use 'overlapping windows' to effectively increase precursor selectivity, then can code it in dia-PASEF like you would in regular DIA, i.e. edit the .txt file which defines the acquisition scheme to put two sets of frames in there instead of one, with the second set shifted by half the window width.

Best,
Vadim

**Comment by joachims1956** [comment]
Hi Vadim, 
Thanks for the fast reply. 
Also using "--tims-scan" does not enable analysis in DIA-NN.

Yes, we to use 'overlapping windows' to effectively increase precursor selectivity, as also shown by you for slice-PASEF acquisitions. In addition, we want to achieve complete coverage of the precursor cloud. 
In contrast to slice-PASEF, we were not able to analyze the sample with the shown dia-PASEF window design. 
Thanks a lot.

Best, 
Joachim

**Comment by vdemichev** [comment]
How does the log look like without --tims-scan? Same digest analysed with regular dia-PASEF is OK?

**Comment by vdemichev** [comment]
Yes, I guess something prevents DIA-NN from processing correctly. You can actually try XICs + DIA-NN Viewer to see how DIA-NN sees the chromatograms. In general, such poor performance is quite strange, as DIA-NN is fine with even some DDA data.

About the scheme: for DIA-NN to deconvolute it best, I would suggest to 'overlap' thing a bit differently. What is required to make it work well 'Slice-PASEF'-style:
- Cycle is split into sub-cycles, each subcycle starts with an MS1 scan.
- The precursor ion cloud coverage of each subcycle is the same.
- There are no overlaps within a subcycle.
- Deconvolution is therefore achived by window boundaries changing between subcycles.
Also, with --tims-scan DIA-NN tries to check if the run is Slice-PASEF (which is detected as signficant window overlaps within a single frame, i.e. diagonal-like frames), and if it is not (like here), it then cancels --tims-scan. This is to prevent 'accidental' processing of dia-PASEF data with suboptimal settings, but also means not working here.
